etc/helm/pachyderm/templates/cloudsqlAuthProxy/deployment.yaml

```diff
@@ -39,11 +39,20 @@ spec:
           - "-instances={{ .Values.cloudsqlAuthProxy.connectionName }}=tcp:0.0.0.0:{{ .Values.cloudsqlAuthProxy.port }}"
         ports:
         - containerPort: {{ .Values.cloudsqlAuthProxy.port }}
-        securityContext:
-          # The default Cloud SQL proxy image runs as the
-          # "nonroot" user and group (uid: 65532) by default.
-          runAsNonRoot: true
         resources: {{ toYaml .Values.cloudsqlAuthProxy.resources | nindent 10 }}
+      {{- if .Values.global.securityContexts.enabled }}
+        securityContext:
```

Comment:
I would suggest something like this for both pod and container securitycontext.
https://gitea.com/gitea/helm-chart/src/commit/dd702a7ae62511c99e4aa3db29421b68f5ea44e3/templates/gitea/statefulset.yaml#L68
I expect we will need to add/remove few values along the way.

Reply:
If not, `runAsUser`, `runAsGroup` and `runAsNonRoot` needs to be on container and Pod level. I know its not required from K8s POV but thats a requirement in our OPA.